user-creation/src/main/java/no/unit/nva/useraccessservice/usercreation/PersonInformation.java

```diff
@@ -28,4 +28,5 @@ public interface PersonInformation {
 
     void setPersonAffiliations(List<PersonAffiliation> affiliationInformation);
 
+    Optional<URI> getPersonRegistryId();
```

Comment:
Optional because in case the Person does not exist the field in the response will be empty.